### DISCUSSION TITLE
github actions + status checks - stuck on pending

### DISCUSSION THREAD

**Discussion by sharonamr** [community/community#44086] [Actions]
### Select Topic Area

Bug

### Body

Hi everyone, could really use your help:
I'm running some gh action job that is updating visuals (from visual tests) and pushing it to the pr branch, like this:
```
    - uses: actions/checkout@v3
      with:
        fetch-depth: 0
        ref: ${{ github.event.pull_request.head.ref }}

    ....more steps here....

    - name: Push visuals changes
      run: |
        git config user.name github-actions
        git config user.email github-actions@github.com
        git add . && git commit -m "Updating visuals"
        git push
```

This works fine, except that i have another workflow which is a required status check, that after the push changed to pending and isn't triggered (`Expected — Waiting for status to be reported`), since I guess when pushing with github actions it skips CI.
I would expect that it will mark the status check as skipped/valid or at least maybe there is a way to make it run again.
Please help.

**Comment by VilakshanDadwal** [comment]
Please give this a try if it helps - https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
